client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -246,31 +246,16 @@ class EmailProvidersComparison extends React.Component {
 	renderGSuiteDetails( className ) {
 		const { currencyCode, gSuiteProduct, translate } = this.props;
 
-		let title = translate( 'G Suite by Google' );
-		let description = translate(
-			"We've partnered with Google to offer you email, storage, docs, calendars, and more."
-		);
```

Comment:
We will display `G Suite` instead of `G Suite by Google` from now on. The copy will be the same for both G Suite and Google Workspace now.